pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,378 @@
+
+### Understanding Delete Markers
+
+When a delete object operation is performed on an object in a versioning-enabled bucket, it does not delete the object permanently but it creates a delete marker on the object. This delete marker becomes the latest and current version of the object with a new version id.
+
+A delete marker has the following properties:
+
+- A key and version ID like any other object.
+- It does not have data associated with it, thus it does not retrieve anything from a GET request (you get a 404 error).
+- By default, it is not displayed in the Control Panel UI anymore.
+- The only operation you can use on a delete marker is DELETE, and only the bucket owner can issue such a request.
+
+To permanently delete an object, you have to specify the version-id in your delete object request.
+
+> [!warning]
+> By default, OVHcloud Object Storage does not replicate delete markers nor replicate the permanent deletion to destination buckets. This behavior protects your data from unauthorized or unintentional deletions.
+
+However, you can still replicate delete markers by adding the DeleteMarkerReplication element to your replication configuration rule. DeleteMarkerReplication specifies if delete markers should or should not be replicated (when versioning is enabled, a delete operation is performed on an object it does not actually delete the object but it flags it with a delete marker)
+
+```json
+{
+  "Role": "string",
+  "Rules": [
+    {
+      ...
+      "DeleteMarkerReplication": {
+        "Status": "Enabled"|"Disabled"
+      }
+    }
+  ]
+}
+```
+
+### Checking the Replication Status
+
+The replication status can be used to determine the status of an object that is being replicated. When you request a source object (using `GET object`) or source object metadata (using `HEAD object`), OVHcloud Object Storage returns the replication status via the header `x-amz-replication-status`.
+
+> [!warning]
+> The replication status only applies to objects that are eligible for replication
+
+The x-amz-replication-status can have the following values:
+
+| Source object                                          | Replica Object                                      |
+|--------------------------------------------------------------|--------------------------------------------------------------|
+| COMPLETED | REPLICA |
+| FAILED | n/a as the replica doesn't exist |
+| PENDING | n/a as the replica doesn't exist yet|
+
+> [!warning]
+> When you replicate objects to multiple destination buckets, the value of x-amz-replication-status is COMPLETED only when the source object has been successfully replicated to all the destination buckets, otherwise, the header remains at the PENDING value.
+> If one or more destination fail replication, the value of the header becomes FAILED.
+
+#### Examples of replication configuration
+Simple replication between 2 buckets
+```json
+{
+  "Role": "IAM-role-ARN",
+  "Rules": [
+    {
+      "Status": "Enabled",
```

Comment:
attribute table should be respected to stay coherent: 
- add "Priority": 1
- add "Filter": { }
- add "DeleteMarkerReplication": { "Status": "Disabled" }